high_availability_guide/_topics/installation.md

```diff
@@ -36,7 +36,14 @@ the non-database {{ site.data.product.title_short }} appliances.
 
     1.  Configure the hostname by selecting **Set Hostname**.
 
-    2.  Select **Configure Database**.
+    For standard manageiq installations, a pre-sized database disk is already attached, initialized, encryption key generated, database configured with a region, and the application started.
```

Comment:
```suggestion
    For standard {{ site.data.product.title_short }} installations, a pre-sized database disk is already attached, initialized, encryption key generated, database configured with a region, and the application started.
```

Reply:
Also, for proper alignment, if this is supposed to be a part of "1.", then indent this line so it aligns with the C in Configure the hostname.  Same goes for the other lines below.  Otherwise it renders as part of the parent list.

Reply:
Yeah, I'm not sure how we want to format a "Note:/FYI".  I can modify this PR or some of my changes can be edited to format it more consistently.

Reply:
Basically, it's not step 1.  step 2 is a branch point... if using out of the box manageiq, you do the first step 2 and the next step.. alternatively, you follow the previous "step 2" and following steps for cloudforms or other setups where the database disk is not already attached/initialized.

Reply:
Oh i see...then maybe it should be like a 

```
1. stuff
2. depending on blah, do one of the following
  a. if blah blah
    1.
    2.
  b. otherwise if blah blah
    1.
    2.
```

Reply:
Yeah, I can do that.  I didn't see examples following that pattern so I just made as few changes as possible so we can discuss format here.  I don't know if @dehawkins512 has opinions on what format we'd like.

Reply:
For "Note:" I had been converting all the HTML to markdown and using this format in bold:
**Note:**